web/app/themes/dp-timber-theme/components/molecule/Card/card.twig

```diff
@@ -0,0 +1,19 @@
+<div class="card-container mx-auto mt-3">
+	<div class="card text-center">
+		<div class="card-header">
+			Storybook Card
```

Comment:
this should be coming from variable considering when we integrate it with the any of the CMS this will render.

Reply:
Okay Got It